db/migrate/20180316095930_create_companies.rb

```diff
@@ -0,0 +1,20 @@
+class CreateCompanies < ActiveRecord::Migration[5.1]
+  def change
+    create_table :companies do |t|
+      t.string :name
+      t.string :web
+      t.string :email
+      t.string :phone
+      t.string :partnermanager
+      t.string :contact_person
+      t.string :email_contact_person
+      t.string :phone_contact_person
+      t.string :crm
+      t.string :level
+      t.string :picture
+      t.boolean :my_company
```

Comment:
always use default value for boolean attributes
in this case probably false
t.boolean :my_company, default: false